src/picongpu/include/particles/manipulators/IfRelativeGlobalPositionImpl.hpp

```diff
@@ -60,7 +60,7 @@ struct IfRelativeGlobalPositionImpl : private T_Functor
         {
             particleInRange1 = isParticleInsideRange( particle1, globalSuperCellOffset);
         }
-        if( isParticle1 )
+        if( isParticle2 )
         {
             particleInRange2 = isParticleInsideRange( particle2, globalSuperCellOffset);
```

Comment:
QUESTION:
if `particle1` and `particle2` are at the same position, both `particleInRange1` and `particleInRange2`have the same boolean value. Is this intended?